src/components/Button/Button.tsx

```diff
@@ -1,3 +1,78 @@
-import { Button } from "theme-ui"
+/** @jsxRuntime classic /
+/** @jsx jsx */
+import { Button as ThemeUIButton, jsx, SxStyleProp } from "theme-ui"
+import { MouseEvent, ButtonHTMLAttributes, forwardRef } from "react"
+import { shade } from "@theme-ui/color"
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: string
+
+  variant?: string
```

Comment:
From what I understood the variant can only be "plain" or "filled", so the types should reflect that
```suggestion
  variant?: "filled" | "plain"
```